src/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  list-style: none;
+  padding: 0;
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin: 0 2px;
+}
+
+.stars--0 {
+  margin-top: 2px;
+}
```

Comment:
no needs to add margin between line of stars. Just use `background-position: center;` for your stars